sdk/schemaregistry/schema-registry/test/schemaRegistry.spec.ts

```diff
@@ -206,4 +206,28 @@ describe("SchemaRegistryClient", function() {
     assertIsValidSchemaId(foundIdSecondCall);
     assert.equal(foundIdSecondCall?.id, registered.id);
   });
+
+  it("schema with whitespace", async () => {
+    const schema = {
+      name: "azsdk_js_test2",
+      group: testEnv.SCHEMA_REGISTRY_GROUP,
+      serializationType: "avro",
+      content:
+        "{\n" +
+        '  "type": "record",\n' +
+        '  "name": "Test",\n' +
+        '  "fields": [{ "name": "X", "type": { "type": "string" } }]\n' +
+        "}\n"
+    };
+    const registered = await client.registerSchema(schema, options);
+    assertIsValidSchemaId(registered);
+
+    const found = await client.getSchemaById(registered.id, {
```

Comment:
May also want to call getSchema as I think that would call the service, so we would be demonstrating the full behavior (granted we may want to change it).

Reply:
I added it with an explanatory comments.